dev/com.ibm.ws.cdi.extension_fat/fat/src/com/ibm/ws/cdi/extension/apps/invocationContext/MyInterceptor.java

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.cdi.extension.apps.invocationContext;
+
+import static org.junit.Assert.assertEquals;
+
+import java.lang.annotation.Annotation;
+import java.util.Set;
+
+import javax.annotation.Priority;
+import javax.interceptor.AroundInvoke;
+import javax.interceptor.Interceptor;
+import javax.interceptor.InvocationContext;
+
+import com.ibm.ws.cdi.internal.interfaces.CDIUtils;
+
+@Priority(1)
+@Interceptor
+@MyInterceptorBinding
+public class MyInterceptor {
+
+String n = null;
+
+    @AroundInvoke
+    public Object aroundInvokeMethod(InvocationContext ctx) throws Exception {
+        InvocationContextTestServlet.bindingInterceptorRan = true;
+        return ctx.proceed();
+    }
```

Comment:
Call `CDIUtils.getInterceptorBindingsFromInvocationContext()` here and assert it returns `@MyInterceptorBinding`.